### ISSUE TITLE
Fail in `sprawl_overview` notebook (missing dependency?)

### ISSUE THREAD

**Issue by delhomer** [lgervasoni/urbansprawl#3]
The `compute_grid_accessibility` method fails during execution. After reading the source code, it seems that the problem comes from line 96:
```
max_processes = int( ''.join(numbers) )
```
The value of `numbers` is probably not int-compatible (I suspect a null value), as a the log highlights a missing dependency (`psutil`).

**Comment by delhomer** [comment]
That's it, after adding `psutil` to conda env, the process is going further than previously... I will propose a small patch to amend the README file.
